exonum/build.rs

```diff
@@ -76,5 +75,7 @@ fn main() {
 fn rust_version() -> Option<String> {
     let rustc = option_env!("RUSTC").unwrap_or("rustc");
     let output = Command::new(rustc).arg("-V").output().ok()?.stdout;
-    String::from_utf8(output).ok()
+    std::str::from_utf8(&output)
+        .ok()
+        .map(|o| o.split_whitespace().nth(1).unwrap().to_owned())
```

Comment:
Are you sure that `unwrap` is safe in this context? I'd return `None` in this case. Also, `?` could be used to simplify the code:

```rust
let rustc_output = std::str::from_utf8(&output).ok()?;
let version = rustc_output.split_whitespace().nth(1)?;
Some(version.to_owned())
```